src/components/StorageGroupInfo/StorageGroupInfo.tsx

```diff
@@ -0,0 +1,174 @@
+
+    if (valueIsDefined(GroupGeneration)) {
+        storageGroupInfoFirstColumn.push({
+            label: storageGroupInfoKeyset('group-generation'),
+            value: GroupGeneration,
+        });
+    }
+    if (valueIsDefined(ErasureSpecies)) {
+        storageGroupInfoFirstColumn.push({
+            label: storageGroupInfoKeyset('erasure-species'),
+            value: ErasureSpecies,
+        });
+    }
+    if (valueIsDefined(MediaType)) {
+        storageGroupInfoFirstColumn.push({
+            label: storageGroupInfoKeyset('media-type'),
+            value: MediaType,
+        });
+    }
+    if (valueIsDefined(Encryption)) {
+        storageGroupInfoFirstColumn.push({
+            label: storageGroupInfoKeyset('encryption'),
+            value: Encryption ? storageGroupInfoKeyset('yes') : storageGroupInfoKeyset('no'),
+        });
+    }
+    if (valueIsDefined(Overall)) {
+        storageGroupInfoFirstColumn.push({
+            label: storageGroupInfoKeyset('overall'),
+            value: <EntityStatus status={Overall} />,
+        });
+    }
+    if (valueIsDefined(State)) {
+        storageGroupInfoFirstColumn.push({label: storageGroupInfoKeyset('state'), value: State});
+    }
+    if (valueIsDefined(MissingDisks)) {
+        storageGroupInfoFirstColumn.push({
+            label: storageGroupInfoKeyset('missing-disks'),
+            value: MissingDisks,
+        });
+    }
+
+    const storageGroupInfoSecondColumn = [];
+
+    if (valueIsDefined(Used) && valueIsDefined(Limit)) {
+        storageGroupInfoSecondColumn.push({
+            label: storageGroupInfoKeyset('used-space'),
+            value: (
+                <ProgressViewer
+                    value={Number(Used)}
+                    capacity={Number(Limit)}
+                    formatValues={formatStorageValuesToGb}
+                    colorizeProgress={true}
+                />
+            ),
+        });
+    }
+    if (valueIsDefined(Available)) {
+        storageGroupInfoSecondColumn.push({
+            label: storageGroupInfoKeyset('available'),
+            value: formatStorageValuesToGb(Number(Available)),
+        });
+    }
+    if (valueIsDefined(Usage)) {
+        storageGroupInfoSecondColumn.push({
+            label: storageGroupInfoKeyset('usage'),
+            value: `${Usage.toFixed(2)}%`,
+        });
+    }
+    if (valueIsDefined(DiskSpace)) {
+        storageGroupInfoSecondColumn.push({
+            label: storageGroupInfoKeyset('disk-space'),
+            value: <EntityStatus status={DiskSpace} />,
+        });
+    }
+    if (valueIsDefined(Latency)) {
+        storageGroupInfoSecondColumn.push({
+            label: storageGroupInfoKeyset('latency'),
+            value: <EntityStatus status={Latency} />,
+        });
+    }
+    if (valueIsDefined(LatencyPutTabletLog)) {
+        storageGroupInfoSecondColumn.push({
+            label: storageGroupInfoKeyset('latency-put-tablet-log'),
+            value: `${LatencyPutTabletLog} ms`,
```

Comment:
Here and farther lets add spaces between the digits of the number (like it is done in `Available Space` field)